### DISCUSSION TITLE
#deploy error

### DISCUSSION THREAD

**Discussion by HithaishiDesai** [m4mallu/uniquify#8] [Q&A] [answered]
![Screenshot_2022-08-13-06-02-30-817_com android chrome](https://user-images.githubusercontent.com/78928904/184460231-06d53835-c2b1-481d-b02f-c098596dac1d.png)

Sir I have used your repo for string session

**Comment by m4mallu** [comment, marked as answer]
Did you use the session string from Pyrogram V2 session string maker for the same?

If so, this error will be thrown from the bot.
**This repo has not been updated to the pyro V2.** So .. use any` old session string generator` to deploy ...

**Reply by HithaishiDesai** [reply]
Ok Sir Thanks for the help 🙏🏻❤️
